float32[] initial_joint_vec
float32[] final_joint_vec
---
float32[] plan_out
